example/exporter/kubelet-servicemonitor.yaml

```diff
@@ -0,0 +1,20 @@
+# You need to have "--kubelet-object=kube-system/kubelet" as a prometheus-operator arg
```

Comment:
If you have defined `--kubelet-object=kube-system/kubelet` you don't need this `ServiceMonitor` . isn't it?